CHANGES.rst

```diff
@@ -69,14 +69,15 @@ astropy.modeling
 - Significant reorganization of the documentation. [#9078, #9171]
 
 - Add ``Tabular1D.inverse`` [#9083]
-- Significant reorganization of the documentation. [#9078, #9171]
```

Comment:
Removing duplicate change log entry should be a separate PR. Unless this was not intentional?